app/src/main/java/org/p2p/wallet/swap/ui/jupiter/settings/presenter/SwapLoadingSettingsMapper.kt

```diff
@@ -0,0 +1,169 @@
+package org.p2p.wallet.swap.ui.jupiter.settings.presenter
+
+import org.p2p.core.common.DrawableContainer
+import org.p2p.core.common.TextContainer
+import org.p2p.uikit.components.finance_block.FinanceBlockCellModel
+import org.p2p.uikit.components.finance_block.FinanceBlockStyle
+import org.p2p.uikit.components.left_side.LeftSideCellModel
+import org.p2p.uikit.components.right_side.RightSideCellModel
+import org.p2p.uikit.model.AnyCellItem
+import org.p2p.uikit.utils.image.ImageViewCellModel
+import org.p2p.uikit.utils.skeleton.SkeletonCellModel
+import org.p2p.uikit.utils.text.TextViewCellModel
+import org.p2p.uikit.utils.toPx
+import org.p2p.wallet.R
+
+class SwapLoadingSettingsMapper(
+    private val commonMapper: SwapCommonSettingsMapper
```

Comment:
i used inheritance for common logic in mappers
let's use one way or another

Reply:
inheritance is bad practice

Reply:
then i should refactor my mapper to composition too

Reply:
local screen mapper, not required I think